### ISSUE TITLE
Script Installation Failure Due to Missing Packages and Incorrect Python Version 

### ISSUE THREAD

**Issue by hoklims** [Trameurs/DofusFashionistaVanced#16]
I am trying to run a script `configure_fashionista_root.py` in the Debian terminal, and I am encountering several issues:

1. Obsolete or Missing Packages:
    The script attempts to install the following packages: `libmysqlclient-dev`, `mysql-server`, `mysql-client`, `python-pip` and `python-dev`, but the terminal returns `Package '...' has no installation candidate`. It recommends replacing them with alternative packages: `libmariadb-dev-compat`, `libmariadb-dev`, `python3-pip`, and `python-dev-is-python3`. 
   
2. Python Version Issue:
    The script attempts to call `python` and `pip` commands which are not available - I have `python3` and `pip3` installed in my system instead.

Here is an abbreviated version of the error message I see when I attempt to run the script:

    ```shell
    File "/home/hokli/Python-3.9.16/fashionista/./configure_fashionista_root.py", line 159, in main
        call(['python', '-m', 'pip', 'install'] + PIP_PACKAGES_TO_INSTALL)
    ...
    FileNotFoundError: [Errno 2] No such file or directory: 'python'
    ```

Thanks for your help. I look forward to hearing back soon.

**Comment by Trameurs** [comment]
Hello,

I'm working with Amazon Linux so the 'yum' packages_to_install:
```
    'yum': [
        'python-pip', 'sqlite', 'pngcrush', 'ImageMagick', 'mariadb-server',
        'mariadb', 'python-devel', 'mariadb-devel', 'libevent-devel', 'memcached',
    ]
```
I haven't tested with 'apt-get', try to switch the packages to be the same as 'yum', I had to do some trial and error to find packages that were working for me. You can also just install everything manually to try one by one. Normally it should work with any mysql server and client, I personally used mariadb.

For the second problem, replace python and pip to python3 and pip3 or use symbolic links :
`sudo ln -s $(which python3) /usr/local/bin/python`
`sudo ln -s $(which pip3) /usr/local/bin/pip`

Let me know if it works!

**Comment by hoklims** [comment]
Thank you for your prompt and informative reply. I followed your advice to switch package installations from 'apt-get' to 'yum' equivalents and to make symbolic links for Python and pip. This allowed me to resolve the issues I was experiencing with configure_fashionista_root.py.

However, it's worth mentioning that I continue to encounter other difficulties while working with the project. These are not insurmountable, but they do add to the complexity of getting everything set up.

I understand that the software is primarily designed for Amazon Linux, and as someone working on a Windows environment, I'm aware of the constraints this may pose. That being said, I am looking forward to a stable release optimized for Windows.

Once again, thank you for your assistance. I appreciate the work you're doing and hope to see it continue to evolve.

Feel free to close this ticket as my initial issue has been resolved.
